src/main/java/cs2103/v15_1j/jimjim/ui/TodayPaneController.java

```diff
@@ -73,17 +74,36 @@ public void refreshData(){
 		}
 
 		if(showOverdue){
-			rowFactory.showOverdue();
+			rowFactory.addLabel("Overdue", "header");
+			int noOfOverdue = rowFactory.showOverdue();
+
+			if(noOfOverdue == 0){
+				rowFactory.addLabel("Congratulations there are no Overdue Events or Tasks", "no-overdue-label");
+			}
+
+			rowFactory.addLabel("", "");
 		}
 
 		showFloatingTasks();
 	}
 
 	private void showFloatingTasks(){
+		rowFactory.addLabel("Floating Tasks", "header");
 		hasCompleted = rowFactory.showAllFloatingTasks(showCompleted);
 
 		int index = todayGridPane.getChildren().size();
 
+		if(displayList.getFloatingTasksList().size() == 0){
```

Comment:
I think it'll help, like Henry said feedback is useful and that it's not No news is good news. In this case it's only when there are completely no floating tasks and I think it's fine to just include that for now.